packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -966,6 +966,7 @@ class ServiceExtensionManager extends Disposer {
       }
     }
 
+    if (mainIsolate == null) return;
```

Comment:
@jacob314 this seemed to fix the flakes we were seeing. Do you see any problem with this change?

Reply:
seems reasonable as it would crash on the next line anyway without the return.